tools/requirements.txt

```diff
@@ -0,0 +1,5 @@
+aiohttp
+click==7.1.2
+pydantic==1.7.3
+kiwi
+kiwi-boxed-plugin
```

Comment:
These two should specify versions (since 4283653):
```
kiwi==9.23.56
kiwi-boxed-plugin==0.2.15
```
Also, can they now be removed from src/requirements.txt?

Reply:
Yep, good catches, thanks.